### ISSUE TITLE
Dealing with www-prefix in AppID

### ISSUE THREAD

**Issue by slisznia** [google/u2f-ref-code#24]
Will browsers match "www.example.com" against "example.com" AppID?

More context here:
http://stackoverflow.com/questions/27756287/u2f-application-id-facet-id-for-a-web-site

**Comment by juanlang** [comment]
No, they won't. See the FIDO AppID and FacetID Specification v1.0, section
3.1, Processing Rules for AppID and FacetID Assertions:
http://fidoalliance.org/specs/fido-u2f-v1.0-ps-20141009/fido-appid-and-facets-ps-20141009.html#processing-rules-for-appid-and-facetid-assertions
.
